### ISSUE TITLE
(2.3 beta-2) Dial-in participant not shown in userlist

### ISSUE THREAD

**Issue by pandaroma** [bigbluebutton/bigbluebutton#11860]
It is possible to enter a conference with a phone, when dial-in is set up like in 2.2. But the particiant is not shown in the userlist. Also, the participant has no feedback, if the entered PIN was right (but he gets a notification if it's wrong).
I can see this participant in a monitoring tool.

**Comment by ichdasich** [comment]
I can confirm this issue on my instance as well.

**Comment by alangecker** [comment]
can confirm it as well.

Following error in the logs might be related:
```
html5-backend-1_1      | error: UserJoinedVoiceConfToClientEvtMsg: Error: Match error: Unknown key in field waitingForAcceptance
html5-backend-1_1      |     at check (packages/check/match.js:36:17)
html5-backend-1_1      |     at addUserPersistentData (imports/api/users-persistent-data/server/modifiers/addUserPersistentData.js:6:3)
html5-backend-1_1      |     at addUser (imports/api/users/server/modifiers/addUser.js:82:3)
html5-backend-1_1      |     at addDialInUser (imports/api/users/server/modifiers/addDialInUser.js:30:10)
html5-backend-1_1      |     at EventEmitter.handleJoinVoiceUser (imports/api/voice-users/server/handlers/joinVoiceUser.js:36:5)
html5-backend-1_1      |     at EventEmitter.emitAsync (/app/programs/server/npm/node_modules/eventemitter2/lib/eventemitter2.js:463:31)
html5-backend-1_1      |     at MeetingMessageQueue.handleTask (imports/startup/server/redis.js:112:10)
html5-backend-1_1      |     at PowerQueue.runTask (packages/cfs_power-queue.js:596:14)
html5-backend-1_1      |     at PowerQueue.updateThrottleUp (packages/cfs_power-queue.js:402:12)
html5-backend-1_1      |     at PowerQueue.next (packages/cfs_power-queue.js:451:10)
html5-backend-1_1      |     at packages/cfs_power-queue.js:565:12
html5-backend-1_1      |     at Meteor.EnvironmentVariable.EVp.withValue (packages/meteor.js:1234:12)
html5-backend-1_1      |     at packages/meteor.js:550:25
html5-backend-1_1      |     at runWithEnvironment (packages/meteor.js:1286:24)
```

**Comment by alangecker** [comment]
so apparently there is a `waitingForAcceptance` field set, but is not allowed by `addUserPersistentData`
https://github.com/bigbluebutton/bigbluebutton/blob/develop/bigbluebutton-html5/imports/api/users-persistent-data/server/modifiers/addUserPersistentData.js#L5

this got changed within #11722 in 2.3-beta-2 
ping @Tainan404 :)

**Comment by spreeni151** [comment]
Same issue here on my server. Is there something I can do to fix it (e.g. set a permission)? I would like to test beta2 in my office, but the telephone call-in is used frequently there. I guess the users would be confused if phone-callers are not shown in the list.

**Comment by prlanzarin** [comment]
Re-opening this until beta3 lands with the fix (#11912)

**Comment by prlanzarin** [comment]
Fixed in https://github.com/bigbluebutton/bigbluebutton/releases/tag/v2.3-beta-3
